### PR TITLE
Add a page describing all general settings

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: documentation/get-started/settings.en.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+# General settings
+
+In QField, you have some general settings; here you will find further details about them.
+
+## Customize search bar
+
+You can change what you want or not be able to search from the search bar.
+
+- **Features from active layer**
+Returns a list of features from the active layer with matching attributes.
```
Comment (lucienicolier) [inline documentation/get-started/settings.en.md:15]:
```suggestion

Returns a list of features from the active layer with matching attributes.
```

File: documentation/get-started/settings.en.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+---
+title: General settings
+tx_slug: documentation_get-started_settings
+---
+
+# General settings
+
+In QField, you have some general settings; here you will find further details about them.
```
Comment (suricactus) [inline documentation/get-started/settings.en.md:8]: If possible, I would avoid `;` in the first paragraph on a page. It breaks the reading flow.

Comment (m-kuhn) [reply documentation/get-started/settings.en.md:8]: An intro like "the general settings page allows you to control basic appearance and behavior of the app" adds a tiny bit more context.

File: documentation/get-started/settings.en.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+If disabled, the mouse will act as a stylus pen.
+
+## Send anonymized metrics
+
+If enabled, anonymized metrics will be collected and sent to help improve QFiled for everyone.
+
+## Dim screen when idling
+
+Time of inactivity in seconds before the screen brightness get to be dimmed to preserve the battery.
```
Comment (suricactus) [inline documentation/get-started/settings.en.md:93]:
```suggestion
Time of inactivity in seconds before the screen brightness get to be dimmed to preserve the battery. Set to 0 to disable dim screen.
```

**Review by suricactus — APPROVED** [review]
I think we better merge and do gradual improvements than keeping this open forever.

**Comment by m-kuhn** [comment]
I think this would benefit from running through a text checker, there are several typos and grammar issues. I think we better fix these before merging to avoid increasing translator's work.

File: documentation/get-started/settings.en.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+If enabled, pressing the device's volume up key will add a vertex while pressing volume down key will remove the last entered vertex during digitizing sessions.
+
+## Consider mouse as a touchscreen device
+
+If disabled, the mouse will act as a stylus pen.
+
+## Send anonymized metrics
+
+If enabled, anonymized metrics will be collected and sent to help improve QFiled for everyone.
```
Comment (m-kuhn) [inline documentation/get-started/settings.en.md:89]:
```suggestion
If enabled, anonymized metrics will be collected and sent to help improve QField for everyone.
```

Not discoverable with grammarly et al

File: documentation/get-started/settings.en.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+## Map canvas rendering quality
+
+You can choose from three options:
+
+- Best quality
+- Lower quality
+- Lowest quality
+
+A lower quality trades rendering precision in favor of lower memory usage and rendering time.
```
Comment (m-kuhn) [inline documentation/get-started/settings.en.md:129]:
```suggestion
A lower quality trades rendering resolution in favor of lower memory usage and rendering time.
```

File: documentation/get-started/settings.en.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+By switching on "Maximized attribute form" it will maximize the attribute form.
+
+## Fixed scale navigation
+
+When fixed scale navigation is active, focusing on a search result will pan to the feature.
+With fixed scale navigation disabled, it will pan and zoom to feature.
+
+## Show digitizing information
+
+When switched on, coordinate information, such as latitude and longitude, is overlayed onto the map while digitizing new features or usting measure tool.
```
Comment (m-kuhn) [inline documentation/get-started/settings.en.md:64]: My browser flags "usting" as not existant word

File: documentation/get-started/settings.en.md (hunk 5 above)
Comment (suricactus) [reply documentation/get-started/settings.en.md:129]: NOTE if we switch to "resolution" over "precision" we should update on QField.

Comment (m-kuhn) [reply documentation/get-started/settings.en.md:129]: @nirvn ?

**Pull request merged**